### PR TITLE
add org.ricsdev.pathnv

### PR DESCRIPTION
**Pull request by RacimFethallah**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.
  

- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements#control-over-domain
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
This is missing:

> Please describe the application briefly.

File: org.ricsdev.pathnv.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+app-id: org.ricsdev.pathnv
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: pathnv
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```
Comment (hfiguiere) [inline org.ricsdev.pathnv.yml:10]: It's either just x11 or wayland with fallback, not both.

File: org.ricsdev.pathnv.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  - --device=dri
+  - --filesystem=home
+  #- --socket=system-bus
+  #- --share=network
+  #- --talk-name=org.freedesktop.DBus
```
Comment (hfiguiere) [inline org.ricsdev.pathnv.yml:15]:
```suggestion
```

File: org.ricsdev.pathnv.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      - mkdir -p /app/share/applications
+      - mkdir -p /app/share/icons/hicolor/128x128/apps
+      - cp -r bin/* /app/bin/
+      - cp pathnv.desktop /app/share/applications/org.ricsdev.pathnv.desktop
+      - cp icons/hicolor/128x128/apps/org.ricsdev.pathnv.png /app/share/icons/hicolor/128x128/apps/
+      - chmod +x /app/bin/pathnv
+    sources:
+      - type: archive
+        #path: pathnv-1.0.0.tar.gz
+        url: https://github.com/RacimFethallah/PathNv/releases/download/1.0.0%2B1/pathnv-1.0.0.tar.gz
```
Comment (hfiguiere) [inline org.ricsdev.pathnv.yml:33]: so how does with work on multiple architectures since you unpack a binary

File: org.ricsdev.pathnv.yml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    only-arches:
+      - x86_64
+    build-commands:
+      - ls -R
+      - mkdir -p /app/bin
+      - mkdir -p /app/share/applications
+      - mkdir -p /app/share/icons/hicolor/128x128/apps
+      - cp -r bin/* /app/bin/
+      - cp pathnv.desktop /app/share/applications/org.ricsdev.pathnv.desktop
+      - cp icons/hicolor/128x128/apps/org.ricsdev.pathnv.png /app/share/icons/hicolor/128x128/apps/
```
Comment (hfiguiere) [inline org.ricsdev.pathnv.yml:28]:
```suggestion
      - cp -r bin/* /app/bin/
      - install -Dm644 pathnv.desktop /app/share/applications/org.ricsdev.pathnv.desktop
      - install -Dm644 icons/hicolor/128x128/apps/org.ricsdev.pathnv.png /app/share/icons/hicolor/128x128/apps/
```

File: org.ricsdev.pathnv.yml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  #- --socket=system-bus
+  #- --share=network
+  #- --talk-name=org.freedesktop.DBus
+modules:
+  - name: FlutterApp
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - ls -R
```
Comment (hfiguiere) [inline org.ricsdev.pathnv.yml:22]:
```suggestion
```

File: org.ricsdev.pathnv.yml (hunk 3 above)
Comment (hfiguiere) [reply org.ricsdev.pathnv.yml:33]: there is still an aarch64 package being built

File: org.ricsdev.pathnv.yml [hunk 6]
```diff
@@ -0,0 +1,35 @@
+app-id: org.ricsdev.pathnv
```
Comment (bbhtt) [inline org.ricsdev.pathnv.yml:1]: ricsdev.org goes to an empty landing page https://ricsdev.org/lander How do I know you control the website? Can you put something up about this project?

**Pull request closed without merging**